src/Akeneo/Platform/Job/back/ServiceApi/JobInstance/DeleteJobInstance/DeleteJobInstanceCommand.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright 2023 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Akeneo\Platform\Job\ServiceApi\JobInstance\DeleteJobInstance;
+
+final class DeleteJobInstanceCommand
+{
+    /**
+     * @param string[] $codes
+     */
+    public function __construct(
+        public array $codes,
```

Comment:
For me readonly property is mostly for this kind of things (public property on command/query)
```suggestion
        public readonly array $codes,
```